apps/nextjs/src/utils/api.ts

```diff
@@ -1,4 +1,4 @@
-// src/utils/trpc.ts
+// src/utils/api.ts
```

Comment:
```suggestion
```

Can just remove it all together - the comments were mainly for the CLI where the files were spread out all over the place but thats not needed here